Mathlib/CategoryTheory/Limits/ConcreteCategory.lean

```diff
@@ -15,7 +15,7 @@ import Mathlib.Tactic.ApplyFun
 -/
 
 
-universe w v u
+universe w v u t r
```

Comment:
```suggestion
universe t w v u r
```
Changing the order of the universes would allow you to write `Concrete.colimit_exists_of_rep_eq.{t}` instead of `Concrete.colimit_exists_of_rep_eq.{_, _, _, t, _}` when `t` has to be given explicitly.

Apart from this, this PR looks good to me.

Reply:
Thanks for the suggestion.